package.json

```diff
@@ -10,6 +10,7 @@
     "build": "tsc",
     "dev": "nodemon --exec ts-node src/index.ts",
     "start": "node dist/index.js",
+    "prestart": "npm run build",
```

Comment:
with this it hooks into the start command and builds the dist folder before running npm start